settings/server/methods.js

```
@@ -103,7 +103,7 @@ Meteor.methods({
       const message = `Update mail configuration: ${error}`;
 
       // Show an error message
-      
+      console.log(message);
```

Instead of `console.log`, we should throw an error. Something like: `return new Meteor.Error(message);`